src/gravitational_waves/gw.cpp

```diff
@@ -51,98 +78,184 @@ GravitationalWave::~GravitationalWave()
 {
 }
 
+double GetHtauSH(const double HR, const double K_sw)
+{
+  return 2. / std::sqrt(3) * HR / std::sqrt(K_sw);
+}
+
+double GetHtauSW(const double HR, const double K_sw)
+{
+  return min(GetHtauSH(HR, K_sw), 1.);
+}
+
+double GetYpsilon(const double HR, const double K_sw)
+{
+  return 1. - 1. / sqrt(1. + 2. * GetHtauSW(HR, K_sw));
+}
+
 double GravitationalWave::CalcEpsTurb(double epsturb_in)
 {
   if (epsturb_in == -1)
   {
-    double HtauSW = 2. / std::sqrt(3) * data.HR / std::sqrt(data.K_sw);
-    return std::pow((1 - std::min(HtauSW, 1.)), 2. / 3.);
+    return std::pow((1 - GetYpsilon(data.HR, data.K_sw)), 2. / 3.);
```

Comment:
```suggestion
    return std::sqrt(1 - GetYpsilon(data.HR, data.K_sw));
```
.. to match our Eq. (3.82).

Reply:
Correct!